libs/cli/langgraph_cli/config.py

```diff
@@ -37,10 +39,13 @@ def validate_config(config: Config) -> Config:
     )
 
     if config.get("node_version"):
-        if config["node_version"] not in ("20",):
+        if config["node_version"] not in SUPPORTED_NODE_VERSIONS:
+            supported = ", ".join(
+                f'`node_version: "{v}"`' for v in SUPPORTED_NODE_VERSIONS
```

Comment:
If it were me I'd keep it a little bit more general and just disallow minor version pins rather than looking for specific values

Not sure how often this gets updated or if it'd be out of sync with the main LangGraph instances

Reply:
just to confirm this is reading version from package.json or from langgraph.json?

Reply:
package.json, but since we already had a check for `node_version == "20"` i made a constant for supported versions and used in both places.

Reply:
Going to just assert:
- is only a major version (no minor or patch) if provided
- Is >= 20